src/screens/ImportSeedPhraseSheet.js

```diff
@@ -247,18 +254,38 @@ export default function ImportSeedPhraseSheet() {
     if (!wasImporting && isImporting) {
       startAnalyticsTimeout(async () => {
         const input = resolvedAddress ? resolvedAddress : seedPhrase.trim();
+        const previousWalletCount = !isObjectEmpty(wallets)
+          ? Object.keys(wallets).filter(
```

Comment:
This is doing a filter on keys, but keys do not have the wallet properties, so this will always return all the keys since undefined never equals readOnly. Can just use lodash filter on wallets

Reply:
Can replace with:
`const previousWalletCount = filter(wallets, wallet => wallet.type !== WalletTypes.readOnly).length`
as lodash filter will handle nil / empty objects as well